.github/workflows/scheduled-build.yml

```diff
@@ -17,13 +17,9 @@ jobs:
         with:
           java-version: 11
       - name: 'Build & Test'
-        env:
-          BRANCH_NAME: ${{ matrix.branch }}
-        run: ./gradlew check -PbranchName=$BRANCH_NAME --info
+        run: ./gradlew check -PbranchName=refs/heads/${{ matrix.branch }} --info
```

Comment:
As far as I can see, the property `branchName` isn't even used in `check`. Maybe we should delete?